napari/_qt/widgets/qt_dims.py

```diff
@@ -130,7 +130,7 @@ def _update_display(self, event=None):
             if axis in self.dims.displayed or nsteps[axis] <= 1:
                 # Displayed dimensions correspond to non displayed sliders
                 self._displayed_sliders[axis] = False
-                self.dims.last_used = None
+                self.dims.last_used = 0
```

Comment:
For typing reasons we now have this always be an `int` instead of sometimes be `None`. I think it's better anyway. Could be reverted if people really wanted but I prefer this behavior